articles/active-directory/manage-apps/disable-user-sign-in-portal.md

```diff
@@ -33,6 +33,10 @@ It's easy to disable an enterprise application so no users can sign in to it in
 1. On the ***appname*** - **Properties** pane, select **No** for **Enabled for users to sign-in?**.
 1. Select the **Save** command.
 
+## Disable unlisted Enterprise apps
```

Comment:
Suggested title: "Use Azure AD PowerShell to disable an unlisted app"

Reply:
done!